rust/operator-binary/src/main.rs

```diff
@@ -58,8 +59,8 @@ async fn main() -> anyhow::Result<()> {
     let opts = Opts::parse();
     match opts.cmd {
         Command::Crd => {
-            SupersetCluster::print_yaml_schema(built_info::PKG_VERSION)?;
-            DruidConnection::print_yaml_schema(built_info::PKG_VERSION)?;
+            v1alpha1::SupersetCluster::print_yaml_schema(built_info::PKG_VERSION)?;
+            druidconnection::v1alpha1::DruidConnection::print_yaml_schema(built_info::PKG_VERSION)?;
```

Comment:
note: Don't use the versioned structs directly here, but instead use the merge functionality described [here](https://stackabletech.github.io/operator-rs/stackable_versioned_macros/attr.versioned.html#kubernetes-specific-features). The code will look similar to this:

```suggestion
            SupersetCluster::merged_crd(SupersetCluster::V1Alpha1)?.print_yaml_schema(built_info::PKG_VERSION, SerializeOptions::default())?;
            DruidConnection::merged_crd(DruidConnection::V1Alpha1)?.print_yaml_schema(built_info::PKG_VERSION, SerializeOptions::default())?;
```

Also take a look at this: https://github.com/stackabletech/opa-operator/blob/e17686ac790faaaf69ed041b6f19109e31f44fd7/rust/operator-binary/src/main.rs#L60-L63

Reply:
https://github.com/stackabletech/superset-operator/pull/597/commits/21082f3297d85b287bab3468a056b1277b3dfbb8